core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/workspace/runners/CloneWorkspaceContainerRunner.scala

```diff
@@ -0,0 +1,120 @@
+package org.broadinstitute.dsde.rawls.monitor.workspace.runners
+
+import bio.terra.workspace.model.{CloneControlledAzureStorageContainerResult, CloneWorkspaceResult, JobReport}
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.rawls.dataaccess.{GoogleServicesDAO, SamDAO, SlickDataSource}
+import org.broadinstitute.dsde.rawls.dataaccess.slick.{WorkspaceManagerResourceJobRunner, WorkspaceManagerResourceMonitorRecord}
+import org.broadinstitute.dsde.rawls.dataaccess.slick.WorkspaceManagerResourceMonitorRecord.{Complete, Incomplete, JobStatus}
+import org.broadinstitute.dsde.rawls.dataaccess.workspacemanager.WorkspaceManagerDAO
+import org.broadinstitute.dsde.rawls.model.{RawlsRequestContext, Workspace}
+import org.joda.time.DateTime
+
+import java.util.UUID
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success, Try}
+
+class CloneWorkspaceContainerRunner(
+  samDAO: SamDAO,
+  workspaceManagerDAO: WorkspaceManagerDAO,
+  dataSource: SlickDataSource,
+  gcsDAO: GoogleServicesDAO
+) extends WorkspaceManagerResourceJobRunner
+    with LazyLogging {
+
+  override def apply(
+    job: WorkspaceManagerResourceMonitorRecord
+  )(implicit executionContext: ExecutionContext): Future[JobStatus] = {
+
```

Comment:
I wasn't considering it to be important, since the job types are mapped up-stream, to give us more flexibility in terms of enabling/disabling, multiple runners on a job, etc. 

But this probably _should_ explicitly fail on any _other_ type of job..... 

I'm convinced: I'll put in a check.

I'm thinking of having it throw an exception explicitly, since 
1. I think that counts as exceptional behavior (not an expected failure case)
2. there's logic in the monitor that logs things in a way that seems correct for this case